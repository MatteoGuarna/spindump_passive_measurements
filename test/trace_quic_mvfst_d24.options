--format text --report-packets --report-notes