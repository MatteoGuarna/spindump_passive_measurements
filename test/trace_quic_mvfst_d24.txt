A simple run with the mvfst stack using the version 0xfaceb001 which corresponds to IETF QUIC draft 24. 