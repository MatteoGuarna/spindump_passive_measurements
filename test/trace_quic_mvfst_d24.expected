QUIC 127.0.0.1 <-> 127.0.0.1 087bb62b0a47c8d6-7ab62b0a47c8d651 (49032:9999) at 1585245386799227 new starting packets 0 0 bytes 0 0 note "No rsp,v.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 087bb62b0a47c8d6-08400000c5d0c872 (49032:9999) at 1585245386799848 change starting packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 087bb62b0a47c8d6-08400000c5d0c872 (49032:9999) at 1585245386799848 measurement up right 621 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 7bb62b0a47c8d651-08400000c5d0c872 (49032:9999) at 1585245386799967 packet up responder length 165 packets 1 2 bytes 1280 240 bandwidth 1280 240 note "V.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 7bb62b0a47c8d651-08400000c5d0c872 (49032:9999) at 1585245386799987 packet up responder length 800 packets 1 3 bytes 1280 1040 bandwidth 1280 1040 note "V.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 7bb62b0a47c8d651-08400000c5d0c872 (49032:9999) at 1585245386800289 packet up initiator length 75 packets 2 3 bytes 1355 1040 bandwidth 1355 1040 note "V.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 7bb62b0a47c8d651-08400000c5d0c872 (49032:9999) at 1585245386800795 packet up initiator length 113 packets 3 3 bytes 1468 1040 bandwidth 1468 1040 note "V.fb24,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 7bb62b0a47c8d651-08400000c5d0c872 (49032:9999) at 1585245386800906 packet up responder length 74 packets 4 4 bytes 1535 1114 bandwidth 1535 1114 note "V.fb24,no spin"
